src/services/getUserService.js

```diff
@@ -0,0 +1,21 @@
+const { queryByUsername } = require("../repository/getUserDAO");
+
+
+async function getUserByUsername(req, res) {
```

Comment:
I would move the path handling portion of this code to a router file. The stuff to do with req, res, and status codes.